### PR TITLE
refactor(lookup): Versioning-related variables

### PR DESCRIPTION
**Pull request by zharinov**
## Documentation (please check one with an [x])

- [ ] I have updated the documentation, or
- [x] No documentation update is required

## How I've tested my work (please select one)

I have verified these changes via:

- [x] Code inspection only, or
- [ ] Newly added/modified unit tests, or
- [ ] No unit tests but ran on a real repository, or
- [ ] Both unit tests + ran on a real repository

### CODE REVIEW

File: lib/workers/repository/process/lookup/index.ts [hunk 1]
```diff
@@ -369,7 +371,7 @@ export async function lookupUpdates(
       if (versioning.valueToVersion) {
         // TODO #7154
         res.currentVersion = versioning.valueToVersion(res.currentVersion!);
-        for (const update of res.updates || []) {
+        for (const update of res.updates || /* istanbul ignore next*/ []) {
```
Comment (viceice) [inline lib/workers/repository/process/lookup/index.ts:374]: is it impossible to create a test? should we use `!` assertion instead?

File: lib/workers/repository/process/lookup/index.ts [hunk 2]
```diff
@@ -440,7 +442,9 @@ export async function lookupUpdates(
     // Handle a weird edge case involving followTag and fallbacks
     if (rollbackPrs && followTag) {
       res.updates = res.updates.filter(
-        (update) => res.updates.length === 1 || update.updateType !== 'rollback'
+        (update) =>
+          res.updates.length === 1 ||
+          /* istanbul ignore next */ update.updateType !== 'rollback'
```
Comment (viceice) [inline lib/workers/repository/process/lookup/index.ts:447]: same here

**Review by viceice — COMMENTED** [review]
otherwise LGTM

File: lib/workers/repository/process/lookup/index.ts (hunk 1 above)
Comment (zharinov) [reply lib/workers/repository/process/lookup/index.ts:374]: It's unrelated coverage miss, I wanted to refactor it later

**Review by viceice — APPROVED** [review]
